crates/bevy_ui/src/layout/ui_surface.rs

```diff
@@ -209,6 +209,7 @@ impl UiSurface {
             width: taffy::style::AvailableSpace::Definite(render_target_resolution.x as f32),
             height: taffy::style::AvailableSpace::Definite(render_target_resolution.y as f32),
         };
+        self.taffy.enable_rounding();
         for root_nodes in camera_root_nodes {
```

Comment:
```suggestion
        };
        for root_nodes in camera_root_nodes {
```
Looks like a leftover